src/SFA.DAS.EmployerIncentives.Commands/Services/BusinessCentralApi/IBusinessCentralFinancePaymentsService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using SFA.DAS.EmployerIncentives.Abstractions.DTOs.Queries.ApprenticeshipIncentives;
+
+namespace SFA.DAS.EmployerIncentives.Commands.Services.BusinessCentralApi
+{
+    public interface IBusinessCentralFinancePaymentsService
+    {
+        public Task<SendPaymentsResponse> SendPaymentRequestsForLegalEntity(List<PaymentDto> payments);
```

Comment:
Can this be just SendPayments ?  There is nothing in the parameter list that has anything to do with LegalEntities. The legal entity filtering has already happened elsewhere so it doesn't make sense to include it on the method name.

Reply:
done